packages/p2p/src/components/app.jsx

```diff
@@ -31,11 +31,11 @@ const App = observer(props => {
         order_id,
         poi_url,
         server_time,
-        setOrderId,
         should_show_verification,
         websocket_api,
     } = props;
     general_store.setAppProps(props);
+    general_store.setOrderId(props.order_id);
```

Comment:
Do we need to `setOrderId`? Why not use a `computed` for `order_id` ?

Reply:
Since the stores are a bit messy at the moment I went with this, in the future we'd want to move away completely from `setAppProps` so this will help us do so.